build.gradle

```diff
@@ -10,16 +10,14 @@ buildscript {
 
     }
     dependencies {
-        classpath Libs.com_android_tools_build_gradle
-        classpath Libs.google_services// Google Services plugin
-        //classpath "com.github.dcendents:android-maven-gradle-plugin:$mavenPluginVersion"
-        classpath Libs.agcp// Huawei Push Plugin
-        classpath Libs.kotlin_gradle_plugin
-        classpath Libs.sonarqube_gradle_plugin
-        classpath 'com.google.firebase:firebase-crashlytics-gradle:2.8.1'
+        classpath("com.google.firebase:firebase-crashlytics-gradle:2.8.1")
```

Comment:
@william-ct why do we have crashlytics in our dependencies?
Also why isn't `detekt` library part of the version catalog?

Reply:
The sample app uses the crashanalytics plugin. The dependency can be moved to the required gradle file as well
I think William decided to keep libraries only used once away from the versions.toml file. Can be changed